<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrincipalWindow</class>
 <widget class="QMainWindow" name="PrincipalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="studentTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortStudentComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteStudentButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertStudentButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editStudentButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="editGroupButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertGroupButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QTableView" name="groupTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sortGroupComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteGroupButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="editTeacherButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertTeacherButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QTableView" name="teacherTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sortTeacherComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteTeacherButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="editPredmetButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertPredmetButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QTableView" name="uchebniPlanTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sortPredmetComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deletePredmetButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="editLessonButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertLessonButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QTableView" name="lessonsTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sortLessonComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteLessonButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="editRaspisanieButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertRaspisanieButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QTableView" name="raspisanieTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sortRaspisanieComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteRaspisanieButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableView" name="additionalTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="additionalComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
